projects/1.20/assets/toms-storage/toms_storage/lang/zh_cn.json

```
@@ -0,0 +1,149 @@
+  "tooltip.toms_storage.inventory_cable_connector": "将容器接入存储网络。\\也可提供网络访问权。\\连接终端需要本方块。",
+  "tooltip.toms_storage.filtered": "可设置过滤",
+  "tooltip.toms_storage.trim": "将容器桥接至存储整合器。",
+  "tooltip.toms_storage.crafting_terminal": "访问与之相邻的容器。\\自带合成栏。",
+  "tooltip.toms_storage.storage_terminal": "访问与之相邻的容器。",
+  "tooltip.toms_storage.wireless_terminal": "延长针对终端的触及距离。\\范围：%d格",
+  "tooltip.toms_storage.paint_kit": "潜行右击方块以设定伪装喷涂目标\\右击可喷涂方块以应用喷涂",
+  "tooltip.toms_storage.open_crate": "将其朝向的方块空间视作一个可存取的物品容器。\\注意：存入物将掉落至该方块空间",
+  "tooltip.toms_storage.inventory_connector": "将所有相邻的物品容器整合至自身\\使用存储或合成终端来访问所整合的容器\\使用存储桥接器可拓展连接面。",
+  "tooltip.toms_storage.inventory_proxy": "可以拓展其指针朝向的方块的存储交互面。",
+  "tooltip.toms_storage.inventory_cable": "用于长距离连接容器。\\存储线缆连接器须紧贴容器放置。\\线缆网络中需至少有一个存储整合器。",
+  "tooltip.toms_storage.inventory_hopper": "将物品导入或导出容器网络\\手持物品右击设置过滤。\\空手Shift+右击清除过滤。",
+  "tooltip.toms_storage.loop": "检测到存储网络中发生了循环",
+  "tooltip.toms_storage.amount": "物品数量：%d",
+  "tooltip.toms_storage.sorting_0": "按物品数量排序",
+  "tooltip.toms_storage.sorting_1": "按物品名称排序",
+  "tooltip.toms_storage.sorting_2": "按模组排序",
+  "tooltip.toms_storage.search_0": "标准搜索",
+  "tooltip.toms_storage.search_1": "自动搜索",
+  "tooltip.toms_storage.search_2": "保持标准搜索",
+  "tooltip.toms_storage.search_3": "保持自动搜索",
+  "tooltip.toms_storage.search_4": "%s同步标准搜索",
+  "tooltip.toms_storage.search_5": "%s同步自动搜索",
+  "tooltip.toms_storage.search_6": "保持%s同步标准搜索",
+  "tooltip.toms_storage.search_7": "保持%s同步自动搜索",
+  "tooltip.toms_storage.ctrlMode_0": "应用能源控制偏好\\拿取单个物品：Shift+右击\\转移一个物品到物品栏：Ctrl+Shift+右击",
+  "tooltip.toms_storage.ctrlMode_1": "精致存储控制偏好\\拿取单个物品：中键点击\\转移一个物品到物品栏：Shift+中键点击",
+  "tooltip.toms_storage.ctrlMode_2": "原版控制偏好\\拿单个物品：右键单击\\转移一个物品到物品栏：Shift+右击",
+  "tooltip.toms_storage.pull_0": "禁止从玩家物品栏中抽取材料参与合成",
+  "tooltip.toms_storage.pull_1": "允许在材料缺失时从玩家物品栏中抽取材料参与合成",
+  "tooltip.toms_storage.filter_item": "过滤：%s",
+  "tooltip.toms_storage.empty": "空",
+  "tooltip.toms_storage.level_emitter": "依据存储网络内容物发出红石信号。",
+  "tooltip.toms_storage.lvlEm_lt_0": "存储数量大于设定值时发信",
+  "tooltip.toms_storage.lvlEm_lt_1": "存储数量小于设定值时发信",
+  "tooltip.toms_storage.adv_wireless_terminal": "Shift+右击终端与之绑定。\\范围：%d格\\终端所在区块须处于加载状态",
+  "tooltip.toms_storage.adv_wireless_terminal.beacon1": "在绑定终端的半径8格内建一个%d级信标，即可在同一维度内任意地方打开终端。",
+  "tooltip.toms_storage.adv_wireless_terminal.beacon2": "建一个%d级信标，即可跨维度维度随处打开终端。",
+  "tooltip.toms_storage.adv_wireless_terminal.bound": "绑定至 (%s, %s, %s)，位于%s",
+  "tooltip.toms_storage.adv_wireless_terminal.keybind": "按下%s键打开",
+  "tooltip.toms_storage.adv_wireless_terminal.keybind.outline": "【%s】",
+  "tooltip.toms_storage.link_public_0": "私有频道",
+  "tooltip.toms_storage.link_public_1": "公共频道",
+  "tooltip.toms_storage.link_remote_0": "访问远程容器",
+  "tooltip.toms_storage.link_remote_1": "供远程访问",
+  "tooltip.toms_storage.items_missing": "某些物品可能未放入合成方格",
+  "tooltip.toms_storage.item_filter": "按多种条件过滤，包括物品、损害值、NBT",
```

```suggestion
  "tooltip.toms_storage.item_filter": "按多种条件过滤，包括物品、耐久度、NBT",
```
不知道扁平化后还有没有损害值这个概念了（）